### DISCUSSION TITLE
Is there any Reason why the Repo for Theila got archieved ?

### DISCUSSION THREAD

**Discussion by LNAhri** [siderolabs/talos#6962] [Q&A] [answered]
Just wondering why the Repo is now a read only archieve without a mention somewhere in the github repo or in some discussions or issues. 

And I also don't see a successor to the "old" UI ? Or is it now preferred to handle everything over the CLI?
https://github.com/siderolabs/theila

Thanks :)

**Comment by frezbo** [comment]
talos is not archived, seems like an issue with github status

**Comment by steverfrancis** [comment, marked as answer]
The Theila repo is archived (not Talos). Theila was archived as we don't have the staff to maintain it in face of competing needs. It needs updating for new versions of Talos (and Sidero Metal.)
Our focus for UI is now on Omni, our SaaS for K8s.
Talos can be fully driven by API or CLI (or Omni.)
